src/System.Private.DataContractSerialization/pkg/System.Private.DataContractSerialization.pkgproj

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <!-- Ideally we'd harvest this from the runtime dependencies -->
+    <Version>4.1.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Implementation only package -->
+    <File Include="$(PlaceholderFile)">
+      <TargetPath>ref/dnxcore50</TargetPath>
+    </File>
+    <File Include="$(PlaceholderFile)">
```

Comment:
Which property will add a placeholder for the compile, but not runtime assets for a framework?